data/xp-pen-artist10s.tablet

```diff
@@ -0,0 +1,28 @@
+# XP-Pen
+# Artist 10S (1st generation)
+#
+
+[Device]
+Name=UGEE 10.1 Tablet Monitor
+ModelName=XP-Pen Artist 10S (1st gen)
+DeviceMatch=usb:5543:004a:UGEE 10.1 Tablet Monitor;usb:5543:004a:UGEE 10.1 Tablet Monitor Pad;usb:5543:004a:UGEE 10.1 Tablet Monitor Mouse
+PairedIDs=
+Class=Bamboo
+Width=8.5
+Height=5.3
```

Comment:
These need to be integers, so please just round them to whatever is closest. It doesn't *really* matter anyway (not sure anything actually uses those)

Reply:
fixed